Network/HTTP2/Arch/Sender.hs

```diff
@@ -288,7 +281,6 @@ frameSender ctx@Context{outputQ,controlQ,encodeDynamicTable,outputBufferLimit}
         fillFrameHeader FrameData datPayloadLen streamNumber flag buf
         off'' <- handleTrailers mtrailers off'
         void tell
-        when (isServer ctx) $ halfClosedLocal ctx strm Finished
```

Comment:
Isn't it good enough to remove this line only?

Reply:
fce595ba83b72f409b93c1ed02776df20f240b39 added `isServer`.
I cannot remember why.